data/data_test.go

```diff
@@ -0,0 +1,132 @@
+package data
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+)
+
+type TestClient struct {
+	*MeetupClient
```

Comment:
Embedding `*MeetupClient` here is not needed.

Reply:
Must have been a vestige of a previous approach, this is moot now anyway.